### DISCUSSION TITLE
Single samples vs Merged samples result

### DISCUSSION THREAD

**Discussion by kpalanikannan** [PGScatalog/pgsc_calc#389] [Q&A] [answered]
For my curiosity, I have merged my samples gVCF files as recommended procedure and calcualted PGS score without ancestry. For same samples gVCF (without merge), I have calcualted with same parameters one by one linear and collected results. 

When I compare the SUM values for same samples between merged approach and unmerged approach, I am getting different SUM score.

What are possible reasons for different scores for same sample? Which approach is correct?

**Comment by smlmbrt** [comment, marked as answer]
This probably has to do with different variants being matched in single samples vs. merged. In the merged samples the non-ref alleles are likely improving matching.

**Reply by kpalanikannan** [reply]
I have same thought as well. In this scenario, would that be always recommended to run as merged to consider non-ref alleles in better manner? 

If merging is better, I have major number of breast cancer negative samples in my merged list, it will create bias on positive samples. I may need to be conscious on choosing the balanced number of positive and negative samples in the merged set.

If I want to study for single sample which has no clue on its negative/positiveness to breast cancer, would that PGS score trusted?

**Reply by smlmbrt** [reply]
I'm not sure, but I don't think the results can be compared so directly. This is part of the reason why we don't currently reccomend using WGS data.
